<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchBarcodeDialog</class>
 <widget class="QDialog" name="SearchBarcodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск товара по штрихкоду</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="ltBarcode">
     <item>
      <widget class="QLabel" name="lblBarcode">
       <property name="text">
        <string>Штрихкод:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtBarcode">
       <property name="inputMask">
        <string>9999999999999; </string>
       </property>
       <property name="maxLength">
        <number>13</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tvProduct">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ltButtons">
     <item>
      <widget class="QPushButton" name="btnSellProduct">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Продать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenConsignment">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Все товары из партии</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="btnBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchBarcodeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchBarcodeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
